### DISCUSSION TITLE
Using with jQuery errors

### DISCUSSION THREAD

**Discussion by Sogl** [alpinejs/alpine#3179] [1. Help]
Today I tried to add Alpine to my Grav CMS project.
Two ways:

### 1. Between `<head></head>`

```twig
{% block javascripts %}
    {% do assets.addJs('jquery', 101) %}
    {% do assets.addJs('https://unpkg.com/alpinejs@3.10.3/dist/cdn.min.js', {loading: 'defer', priority: 99}) %}
{% endblock %}

```

and got much errors:
![image](https://user-images.githubusercontent.com/7556290/193089874-10464be6-dcaf-4705-9ea8-c3604fc2932f.png)

What I see about 1st error `Uncaught SyntaxError: missing ) after argument list` in generated js-file (jQuery + Alpine.js):

![image](https://user-images.githubusercontent.com/7556290/193097344-cf025ece-b121-4d8f-bada-c30fea598b96.png)

### 2. Before `</body>`

```twig
 {% do assets.addJs('https://unpkg.com/alpinejs@3.10.3/dist/cdn.min.js', {group:'bottom', loading: 'defer', 
priority: 99}) %}
```

and also got errors:
![image](https://user-images.githubusercontent.com/7556290/193102060-c5cb0f46-0fc8-4cc5-adac-f5e1a8aeb04e.png)

About `Unexpected end of input`. I have semicolon in file:
![image](https://user-images.githubusercontent.com/7556290/193103626-0921630d-1d87-4b3d-95b5-4b179b718318.png)

But no in Chrome Sources:
![image](https://user-images.githubusercontent.com/7556290/193103880-d3093b89-bbb6-4881-b47a-90af52380339.png)

How to fix?

**Comment by ekwoka** [comment]
Firs tstep is remove jQuery, because it isn't 2015 anymore.

But this is caused by, most likely something in an Alpine expression you made a mistake on.
